cmd/mattermost/commands/config.go

```diff
@@ -36,12 +41,22 @@ var ConfigSubpathCmd = &cobra.Command{
 	RunE: configSubpathCmdF,
 }
 
+var ConfigGetCmd = &cobra.Command{
+	Use:     "get",
+	Short:   "Allow seeing the value of a config setting",
```

Comment:
Minor nit since this matches the text in the ticket, but I think this would sound better as
```
Short: "Get config setting",
Long: "Gets the value of a config setting by its name in dot notation.",
```

Reply:
Done